### PR TITLE
New package: mp3gain-1.5.2

### PR DESCRIPTION
**Pull request by lemmi**

### CODE REVIEW

File: srcpkgs/mp3gain/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'mp3gain'
+pkgname=mp3gain
+version=1.5.2
+revision=1
+_rev=_r2
+wrksrc=${pkgname}-${version}${_rev}
```
Comment (Vaelatern) [inline srcpkgs/mp3gain/template]: Best to actually have the revision information in the version variable.
```
_distver=1.5.2
_rev=r2
version=${_distver}_${_rev}
```
or something like that.

**Comment by lemmi** [comment]
yeah, either way it's messy. you can't have _ in the version string, so need to work around that and the zip doesn't contain a directory.
also i'm a little bit opposed to adding more ".", just to replace them later on. makes it kind of opaque.

**Comment by Vaelatern** [comment]
Doesn't need a `.`. Just needs a way that, when a new tarball is downloaded, for a version change to happen instead of a revision bump.

**Comment by lemmi** [comment]
Let's give this a try then :)

File: srcpkgs/mp3gain/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'mp3gain'
+pkgname=mp3gain
+_distver=1.5.2
+_rev=r2
+version=${_distver}.${_rev}
+revision=1
+wrksrc=${pkgname}-${version}
```
Comment (Vaelatern) [inline srcpkgs/mp3gain/template]: Line no longer needed

**Pull request merged**